inst/scripts/insert_image.R

```diff
@@ -69,9 +79,43 @@ local({
             basename(target_dir), "/", basename(input$target)
           )
           w = input$w; h = input$h; alt = input$alt
-          if (w == '' && h == '') paste0('![', alt, '](', s, ')') else shiny::img(
-            src = s, alt = alt, width = if (w != '') w, height = if (h != '') h
-          )
+          if (w == '' && h == '') {
+            paste0('![', alt, '](', s, ')')
+          } else {
+            if (ctx_ext == "rmd") {
+              check_pandoc_format = function(x) {
```

Comment:
You are basically reinventing `htmltools::validateCssUnit()` in this function :) You can probably use `try(htmltools::validateCssUnit())` and turn the possible error to a warning.

Reply:
Yup, that's pretty much exactly what I did 🤣! I'll update to use `htmltools::validateCssUnit()` rather than reinventing the wheel.